CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
 
-| Date        | Notes                               |
-|-------------|-------------------------------------|
-| May 3, 2023 | Initial release of Ner4Opt library. |
+| Date         | Notes                               |
+|------------- |-------------------------------------|
+| Mar 11, 2024 | Add a new preprocessing routine before extracting entites. |
```

Comment:
you can add Issue link here to make it explict and reveal the context

Reply:
Sounds good, will do that.

Reply:
plz also thank the issue submitter using @ sign